src/Test.FunctionalTests.BinSkim.Rules/RuleTests.cs

```diff
@@ -1338,7 +1338,7 @@ public void BA3003_EnableStackProtector_Fail()
         [Fact]
         public void BA3003_EnableStackProtector_NotApplicable()
         {
-            this.VerifyApplicability(new EnablePositionIndependentExecutable(), new HashSet<string>());
```

Comment:
>EnablePositionIndependentExecutable [](http://example.com/codeflow?start=41&length=35)

I just found this old bug  from 2018, should fix it since   related to our tests efforts :)
I only added 2 new tests, all others are moved from not applicable folder.